### DISCUSSION TITLE
Several Lib002 errors that are constantly reoccurring

### DISCUSSION THREAD

**Discussion by MiladyNotMaam** [cdnjs/cdnjs#14149] [General] [answered]
My issue started with a build in Azure DevOps.  Since that time I am consistently getting Lib002 errors with different libraries in my libman.

I was using VS 2019 and had these issues.  I realized that my IDE needed updates and that seemed to fix the lib002 errors.  I completely deleted my libman and started from scratch and everything was fine.  That was yesterday.  This morning I started getting the errors again.  I started down the same path of libman cache clean and libman restore.  I even deleted it again and did libman init.

Got frustrated and moved to VS 2022.  Still the same thing.
I only have 4 libraries and each of them have given me this problem at one point.  Right now it is jquery-validation-unobtrusive@3.2.12.  I have even tried different versions.  Here is my libman:
{
  "version": "1.0",
  "defaultProvider": "cdnjs",
  "libraries": [
    {
      "library": "jquery@3.6.0",
      "destination": "wwwroot/lib/jquery/"
    },
    {
      "library": "twitter-bootstrap@5.1.3",
      "destination": "wwwroot/lib/twitter-bootstrap/"
    },
    {
      "library": "jquery-validation-unobtrusive@3.2.12",
      "destination": "wwwroot/lib/jquery-validation-unobtrusive/"
    },
    {
      "library": "jquery-validate@1.19.5",
      "destination": "wwwroot/lib/jquery-validate/"
    }
  ]
}

Here is the current error:  LIB002	The "jquery-validation-unobtrusive@3.2.12" library could not be resolved by the "cdnjs" 

I am so frustrated by this inability to resolve perfectly fine libraries with cdnjs at any given time and for no reason that I can tell.  Anyone that can help would be appreciated.  I'm beyond done with this.  Sorry to vent.  I'm usually a pretty calm person. :)

**Comment by MattIPv4** [comment, marked as answer]
👋 This was a known bug with LibMan as a result of recent API changes within cdnjs. Please update your LibMan version and reach out to the LibMan team if you continue to run into errors. cdnjs does not maintain the LibMan integration.

**Reply by MiladyNotMaam** [reply]
Thank you!  It turns out that I was using the latest CLI version but the nuget package that my solution was using was very outdated.  I really appreciate your help.
